users-auth/auth-router.js

```diff
@@ -7,12 +7,25 @@ router.post('/register', async (req, res, next) => {
 	try {
 		const { username, password, phoneNumber} = req.body
 		const user = await Users.findBy({ username }).first()
+        const phonenumber = await Users.findBy({ phoneNumber }).first()
```

Comment:
This line is incorrectly indented.
You can use a tool like [prettier](https://prettier.io/) to fix formatting like this.
Improperly formatted code will cause merge conflicts if it is fixed on multiple branches (which is very likely to happen if additional features are added to this file)

I will not merge this until at minimum this line is properly intended. It would be even better if a formatting tool was setup and used across the backend team to enforce proper formatting.